### ISSUE TITLE
FDroid: App contains ads ?!?

### ISSUE THREAD

**Issue by ironspock** [organicmaps/organicmaps#7157]
Hi Team,

your app description on FDroid is stating:
At Organic Maps, we value privacy:
• No Location Tracking
• No Data Collection
• No Ads

However all recent versions on FDroid are marked with Anti-Feature: This App contains advertising!

So something is wrong here.

Best regards
ironspock

**Comment by Jean-BaptisteC** [comment]
Recently we have implemented a new link when you select hotel to open Kayak website, but it's not accepted by Fdroid, this topic is discussed to find the best way for FOSS community :)

**Comment by Mero79i** [comment]
@Jean-BaptisteC stop lying to audience.

@ironspock, in short, they added referral link to KAYAK that TRACKS you (see [this post](https://github.com/organicmaps/organicmaps/issues/7119) with proofs of tracking) and ignoring users opinions to remove this.

See this discussions:
[Main discussion](https://github.com/organicmaps/organicmaps/pull/6523)

And some issues about this:
[First](https://github.com/organicmaps/organicmaps/issues/6707), [second](https://github.com/organicmaps/organicmaps/issues/6773), [third](https://github.com/organicmaps/organicmaps/issues/6668), [fourth](https://github.com/organicmaps/organicmaps/issues/6967), [fifth](https://github.com/organicmaps/organicmaps/issues/6774)

Most of them closed to avoid them being hated because of their greed.

If you need explanation about why it is greed, just read [this thread](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/14082/) on official F-Droid repo. In short they ignored all users that asked to remove that (or at least make it opt-in) and even closed PR with making it opt-out ([proof](https://github.com/organicmaps/organicmaps/pull/6720)). They have been even making excuses that this is useful link, but there is many open source solutions that don’t track you instead (they rejected request to replace this with open source solution ([proof](https://github.com/organicmaps/organicmaps/issues/6774)))

In short that just want more money and they don’t care about community.

**Comment by Mero79i** [comment]
P.S: if you donating to them, consider stopping. They have enough money from KAYAK referral program (even if they will say they are not)

**Comment by Hchgufts** [comment]
@ironspock IF you think they are poor, please take a look on [this](https://en.m.wikipedia.org/wiki/Maps.me#cite_ref-19).

They can wipe this so I hope you will receive notification on your email :)

**Comment by matkoniecz** [comment]
@troll_deleting_accounts

Maps.me and organic maps is not the same thing. But you are probably aware about this.

**Comment by pastk** [comment]
> 
> However all recent versions on FDroid are marked with Anti-Feature: This App contains advertising!
> 

The F-Droid client gives more detailed information, why this Anti-Feature flag was added:

![Screenshot_20240114-120827_1.jpg](https://github.com/organicmaps/organicmaps/assets/18434508/bb31c1aa-7578-478e-bc7d-dba0f335f8b0)

**Comment by biodranik** [comment]
Unfortunately, FDroid breaks all rules and not only mislabels OM as "contains advertising" instead of "shows a button with an explicitly disclosed referral link for some selected hotels", but also marks the whole app with that label, not only a few selected builds. And also does not clarify anything on the desktop version, which just "contains ads" (see screenshot below).

Retroactively marking OM with that discriminatory and misleading label, without clarifying/updating FDroid rules is unacceptable. So I'm asking @licaon-kter to remove all Organic Maps builds from FDroid repository and the website that are labeled as "contains ads".

<img width="699" alt="image" src="https://github.com/organicmaps/organicmaps/assets/170263/beaf12f6-6463-4c72-b76f-1474a671cb5d">

**Comment by rollsicecream** [comment]
I am just here to re-confirm what was said by @pastk. I also get the F-Droid explanation with the detailed info.

**Comment by licaon-kter** [comment]
@rollsicecream MergeRequests welcomed :)

**Comment by mnalis** [comment]
> Though, I don't know why the F-Droid website doesn't display that info, only their app has that 🤷

I've opened issue to track that:  https://gitlab.com/fdroid/fdroid-website/-/issues/770

AFAIR, that detailed explanation text for AFs is relatively new feature, and it is not yet being displayed by website but only by app (which I agree is unfortunate and should be fixed. Merge requests to implement that are, as already noted, welcome!)

**Comment by mnalis** [comment]
> Unfortunately, FDroid breaks all rules and not only mislabels OM as "contains advertising" instead of "shows a button with an explicitly disclosed referral link for some selected hotels",

That has been discussed [several](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/14082#note_1665887323) [times](https://github.com/organicmaps/organicmaps/pull/6523#issuecomment-1808968781) @biodranik. There is _no mislabeling_ according to any [common dictionary definitions of advertising](https://www.wordnik.com/words/advertising). It seems that point of disagreement is that some people accept only very specific (while admittedly very popular in last years) instances of _"advertising"_ (specifically, only _"intrusive banner ads"_), and refuse to accept any and all other commonly accepted definitions of _"Ads"_.

Perhaps (if one is unwilling to accept common dictionary definitions) it would be best to accept this [suggestion](https://github.com/organicmaps/organicmaps/issues/6668#issuecomment-1826390641) then and change demonstrably untruthful (according to common dictionaries) _"No Ads"_ claim to at least **technically correct** _"No annoying Ads"_ or _"No Banner Ads"_. Or (if one would like to avoid looking like sneaky weasel by using such technically correct but quite misleading claims) _**"Only contains non-intrusive commercial advertisements"**_ (which, while being way more truthful and clear, is also admittedly somewhat less catchy marketing phrase for selling purposes to privacy-oriented crowd). Although just removing that _"No Ads"_ claim would remove confusion in acceptable way too (in a ["claiming the Fifth"](https://www.nolo.com/legal-encyclopedia/your-fifth-amendment-right-against-self-incrimination.html) way).

>  but also marks the whole app with that label, not only a few selected builds

This is untrue, to the best of my knowledge. Only OM versions which contain controversial Kayak ads are marked with that anti-feature.
Check for yourself: https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/app.organicmaps.yml

> Retroactively marking OM with that discriminatory and misleading label,

AFAIK, F-droid folks mark all affected versions with their anti-feature as soon as they become aware of it (and finish the process of  discussing & implement it). Doing anything else (claiming that some versions are not affected by anti-feature while they are in fact affected) would be untruthful and is unwanted for hopefully obvious reasons. Thus, retroactively marking affected versions is a must. 

> without clarifying/updating FDroid rules is unacceptable

Ads anti-feature is explained at https://f-droid.org/en/docs/Anti-Features/#Ads

If you think it is unclear what is meant by the term "[advertisment](https://en.wiktionary.org/wiki/advertisement)" (i.e. _"A commercial solicitation designed to sell some commodity, service or similar"_) and think it needs further clarifications that there exists several **different grades of advertisement _invasiveness_ of Ads** (e.g. _"wait & watch this advertisement video to the end in order to be able to continue"_,  _"pop-over ads"_, _"undismissable banner ads"_, _"dismissable banner ads"_, _"externally sourced affiliate links"_ etc.) feel free to do Merge Request against https://gitlab.com/fdroid/fdroid-website/-/blob/master/_docs/Anti-Features.md#Ads to improve user awareness of the situation (and do mention it here, so other interested parties can offer their collaboration).

**Comment by rollsicecream** [comment]
> > Though, I don't know why the F-Droid website doesn't display that info, only their app has that 🤷
> 
> I've opened issue to track that: https://gitlab.com/fdroid/fdroid-website/-/issues/770
> 
> AFAIR, that detailed explanation text for AFs is relatively new feature, and it is not yet being displayed by website but only by app (which I agree is unfortunate and should be fixed. Merge requests to implement that are, as already noted, welcome!)

Thank you! I was not really comfortable creating a GitLab account just for that issue (I know there's a mirror on GitHub, but you can't create issues/PR)

**Comment by pastk** [comment]
@biodranik don't delete posts please!

If you're certain a post is inappropriate then just "Hide" it.
Then users will be able to expand it and read if they choose too.

**Comment by mnalis** [comment]
@biodranik I agree with @pastk, hiding is the best solution here. 

By deleting them, devs are only giving @ghost ammunition on other platforms to also slap a `"censorship"` label on them :disappointed: (and people are then unable to verify who is in the wrong as comments are deleted)

**Comment by matkoniecz** [comment]
To be clear, there are also benefits of deleting this trash comments from troll who posts low quality comments from multitude of accounts they keep deleting.

**Comment by matkoniecz** [comment]
If I had it on my repo then I would likely archive page at Internet Archive, post link to snapshot and delete comments.

Or hide them so everyone can see their lack of quality.

**Comment by mnalis** [comment]
> Or hide them so everyone can see their lack of quality.

Yes, hiding them seems the best of the available options. It takes about the same amount of screen estate as hidden comments, disk usage is minuscule anyway, and by deleting instead of hiding devs are destroying evidence -- evidence that could've worked in their favor. 
And when destroying evidence, most people will (somewhat reasonably) assume that you did it because you _have something to hide_, and not that you were just doing regular spring cleaning that just happened to coincide with heated debate. (_Especially_ if someone gives them a hint claiming so -- like I guess @ghost is doing as we speak).

If the @ghost's  comments (_if_ theirs -- we can't tell now, can we?) were just hidden instead of deleted, everyone interested could expand them and see if that's just another internet troll abusing throwaway accounts, or something else. 

But when deleted -- then nobody could say what was said and by who:
- it could've been the troll trolling with its throwaway sockpuppets, 
- it could've been real insightful comments by real people like you or me, but which are pointing to mismatch between OM advertisement claims and reality, and thus OM is deleting them to protect the image they're trying to project to the public 
- it could even be huge [EdwardSnowden](https://en.wikipedia.org/wiki/Edward_Snowden)-alike whistleblower reveals which disclosed horrible and illegal things being done by OM leadership! [^1]

But since the comments were _deleted_ instead of _hidden_, one can only guess which of those options is the truth.

---

So, to conclude, with best interest of OM in mind: please only **hide** ghost's comments instead of deleting them.

[^1]: I am **not** implying that there _are_ horrible illegal things being done by OM, nor that the @ghost is anything like mr. Snowden. Quite to the contrary, in fact, @biodranik's promise [here](https://github.com/organicmaps/organicmaps/issues/6967#issuecomment-1890939017)  gives me some hope that they are working on solution to rectify advertisement  issues brought by Kayak integration. But I want to show why **deleting** comments ***instead*** of **hiding** them is bad idea for OrganicMaps devs

**Comment by matkoniecz** [comment]
> by deleting instead of hiding devs are destroying evidence 

I would not call it "destroying evidence" - that has some specific legal connotations and is (often?) outright criminal and illegal

> disk usage is minuscule anyway

Microsoft anyway keeps deleted ones, and Microsoft pays for it anyway so it is not a real concern :)


Also, note that deleted comments were engaging in mendacious trolling in ways not entirely obvious for someone unfamiliar with situation and could mislead people. So I see why they were deleted. (and yes, it was mendacious trolling from throwaway self-deleted accounts)

**Comment by no-name008** [comment]
@biodranik i came from [PrivacyGuides forum discussion ](https://discuss.privacyguides.net/t/organic-maps-adding-kayak-referral-links-to-mobile-app/15382). 

I’m not a hater, nor your user. I tried your app and it is definitely looking like advertising. So I don’t think that AF is not correct. It is not annoying, but it is still an ad.

And this totally looks like you are censoring your users and don’t care about their opinion, ignoring requests that will make it togglable. I hope you will change your mind, but for now (IMHO) you are the one who should be blamed for this whole situation, not “ghosts” (who?) or someone else.

**Comment by matkoniecz** [comment]
And this issue had some unhelpful and/or trolling and/or confused comments from user which then immediately deleted account.

There was similar activity earlier that looks like single malicious person wasting their time.

**Comment by mnalis** [comment]
> I would not call it "destroying evidence" - that has some specific legal connotations and is (often?) outright criminal and illegal

You perhaps wouldn't, but you can be almost sure that @ghost puppetmaster troll will try to portray it that way, which is the **risk** associated with **_deleting_** comments instead of just _hiding_ them (that I tried to emphasize and suggest avoiding). 

Also, trying to use other (more polished) words to more objectively present it in roundabout way _might_ end up looking like _"trying to [weasel out](https://idioms.thefreedictionary.com/weasel+out)"_, thus it might be preferable for people to see it in plain language.

Much like _"we do not have advertisements in OrganicMaps app, we just non-intrusively promote specific external service for financial gain"_ (paraphrasing what seems to be publicly visible OrganicMaps leadership stance as far as I understand it) might be seen like _"trying to weasel out"_ to many (esp. privacy-oriented/Ads-hating) people.

> Microsoft anyway keeps deleted ones, and Microsoft pays for it anyway so it is not a real concern :)

perhaps, but I don't have the proof of them keeping deleted ones, and I don't know if OM is paying for their account or using free one,  and if the amount of disk space usage is limited or influences the price. In any case I concluded it would be irrelevant, so I did try to avoid dwelling on it and making my post even bigger than it is. :smile_cat: 

> Also, note that deleted comments were engaging in mendacious trolling in ways not entirely obvious for someone unfamiliar with situation and could mislead people. So I see why they were deleted. (and yes, it was mendacious trolling from throwaway self-deleted accounts)

So, you've actually seen those comments before they got deleted? I didn't, so I could only _guess_ that there is some likelihood that they they might've been written by that @ghost troll, and even less about what they were saying (just because they are anti-social in their behavior, does not mean their arguments are worthless - while it may be strongly correlated, it would be an [ad hominem  logical fallacy](https://en.wikipedia.org/wiki/Ad_hominem) to reject an argument based on reputation of person presenting it).

In fact, while I **strongly disprove** of the way the @ghost troll is behaving (throwaway accounts, issue spamming, sock-puppetry, rude language, childish tantrums etc), I do think that bringing up to the public the fact that apps _failed to deliver_ on its promises of _"No Ads"_ was **good idea**, and that _Ads anti-feature_ in f-droid as a result of that was a **benefit** to pro-privacy/anti-ads crowd. And that even bigger benefit would be if devs (as a result of that hubbub) [manage to deliver](https://github.com/organicmaps/organicmaps/issues/6967#issuecomment-1890939017) the privacy-friendly/no-ads solution as the result, and consider such pro-user-privacy issues as more important than mere technical improvements in the future. (especially due to niche market that OM specifically targets: after all, if privacy/ads etc. are unimportant for some group of people, they may as well continue using MAPS.ME or GoogleMaps)

Anway, if you @matkoniecz _personally_ did see those messages (account names and their content) before they got deleted (as I didn't), then _I_ can accept that claim that they were written by that specific troll and they did not contain new useful information -- but only because _I_ do trust you personally (as a result of multi-year familiarity with your work). However, if that same claim was made by OrganicMaps devs (or any other random person I do _not_ have a multi-year trustful experience with), I would **not** be likely to trust such claims without further evidence (nor _should_ anyone, IMHO).

**Comment by matkoniecz** [comment]
> So, you've actually seen those comments before they got deleted?

yes, I then made https://github.com/organicmaps/organicmaps/issues/7157#issuecomment-1890473645 comment (with "EDIT" segment added after these comments were deleted)

> does not mean their arguments are worthless

oh yes, that is why I am additionally irritated at that person ("Also this kind of trolling is most effective as poisoning discussion on this topic") as I would consider such referral links to be in fact an ad. But their methods are most effective on derailing discussion and ensuring that nothing productive will be done on this topic.

**Comment by pastk** [comment]
> comments from multitude of accounts they keep deleting.

> (and yes, it was mendacious trolling from throwaway self-deleted accounts)

Is there an evidence to support those accounts were self-deleted indeed?

Actually the ghost never hid that all (or the most of) ghost accounts are theirs and they're being suspended by Github.
E.g. see https://github.com/organicmaps/organicmaps/pull/6523#issuecomment-1832584842 (they did mention it in several posts actually).

I'm not defending them, but I'd like to avoid accusations if they're ungrounded (are they?).

**Comment by fitojb** [comment]
> I'm not defending them

Breaking news: you are.

**Comment by mnalis** [comment]
>>    I'm not defending them

> Breaking news: you are

Um, are you sure @fitojb ? I'm personally not even clear who `them` are in that @pastk message:
- the OM devs / project owners (e.g. _"I'm not defending the devs for engaging in censorship by deleting comments they disagree with"_)
- the @ghost accounts (e.g. _"I'm not defending the person who annoyingly created multiple accounts and issue-spammed everyone to complain about advertisements in supposedly NoAds app"_)
- GitHub owners  (e.g. _"I'm not defending Microsoft for suspending / removing those accounts using AI without due process"_)
- someone else? :shrug: 

Anyway, I'd much prefer if devs would (in the future, eh? It's obviously [No Use Crying Over Spilt Milk](https://writingexplained.org/idiom-dictionary/dont-cry-over-spilt-milk)) just hide offending comments, and add a short comment explaining why those comments were hidden; instead of deleting them (without explanation). That way, everyone interested enough could easily decide to unhide the comments, and check for themselves which side is being (more) deceptive. Trust is built by being transparent.

For me personally, I don't think either side is completely without blame, but I'm still waiting to see if there will be some positive pro-user-privacy  result from [this promise by OM devs](https://github.com/organicmaps/organicmaps/issues/6967#issuecomment-1890939017) , so OrganicMaps can be reinstated in F-droid app store.

**Comment by muralito** [comment]
> For me personally, I don't think either side is completely without blame, but I'm still waiting to see if there will be some positive pro-user-privacy result from [this promise by OM devs](https://github.com/organicmaps/organicmaps/issues/6967#issuecomment-1890939017) , so OrganicMaps can be reinstated in F-droid app store.
I think that from the privacy point of view, is its much better to use OM from fdroid, including or not that is is tagged with AF,  than to use it from the PlayStore. Only one of my devices is running android with google propietary parts, the others are running lineageOS so i dont't use PlayStore on them, but obviously i want to keep updating OrganicMaps.

**Comment by biodranik** [comment]
> I think that from the privacy point of view, is its much better to use OM from fdroid

That is not true. The only difference from the privacy point of view is in the presence of Google Location services library that provides faster location detection for users and can be turned off completely from OM settings if not needed.

**Comment by licaon-kter** [comment]
@biodranik Play store means Google services running and tracking you... (from the privacy point of view lol)

**Comment by biodranik** [comment]
@licaon-kter are you sure that google code is running and tracking you if it is not called/used by the app/disabled in settings? Can you prove that?

**Comment by licaon-kter** [comment]
I think Google has lost at least 2 trials regarding "location tracking abuse" already. Google system code is proprietary, I/you can't be sure of anything.

Maybe it sends your location to Google every 4 minutes, sisce it connects to Google so often, who knows? Why not just be on the safe side and not have Play and don't worry about that $hit? I guess that's what the user above wanted.

**Comment by licaon-kter** [comment]
@biodranik not sure what you talk about, I never said nor implied that **OM sends anything**, please don't construct straw-mans nor fight them :smile: 

My posts above are **about Play** itself :shrug: 

The user above was clear in their wish
> Only one of my devices is running android with google propietary parts, the others are running lineageOS so i dont't use PlayStore on them, but obviously i want to keep updating OrganicMaps.

**Comment by muralito** [comment]
> @biodranik not sure what you talk about, I never said nor implied that **OM sends anything**, please don't construct straw-mans nor fight them 😄
> 
> My posts above are **about Play** itself 🤷
> 
> The user above was clear in their wish
> 
> > Only one of my devices is running android with google propietary parts, the others are running lineageOS so i dont't use PlayStore on them, but obviously i want to keep updating OrganicMaps.

That is my point. The big problem for privacy is not OM itself but the playstore and related G services, if OM isn't available in FDroid, the users only have the PlayStore option.

**Comment by muralito** [comment]
> Please stop guessing, that is not professional. Did anyone already check if the Google Play version of Organic Maps with disabled Google Play Location services sends some data to Google? If the setting in OM is disabled, Google's library code is not used at all. Anyone can sue Google if they run the code even if developers are not using it.
> 
> Proofs, please. No guessing.

As said before, i'm not guessing, the problem is PlayStore, and not the OM version of PlayStore. You can distribute in the PlayStore exactly the same version of OM that is in FDroid, byte by byte, but the privacy issue comes from the fact that you need to use the PlayStore and related services.

**Comment by biodranik** [comment]
You are not forced to use Play Store, and can always install the latest Google-built apk directly. 

FYI, we have spent days if not weeks on fixing some ungoogled firmware issues related to location services that reproduced _only in FDroid builds_ that _did not_ contain the Google Location library linked with the apk.

I can not 100% prove that using OM linked with GP location services doesn't do anything bad in the background, and would be interested to hear it from someone who has time and can monitor outgoing network connections, for example.

**Comment by muralito** [comment]
> You are not forced to use Play Store, and can always install the latest Google-built apk directly.
Maybe I can , and maybe also build the apk myself, but probably it is out of reach for a lot of users, and I lose the benefits of using a repo.
 
> FYI, we have spent days if not weeks on fixing some ungoogled firmware issues related to location services that reproduced _only in FDroid builds_ that _did not_ contain the Google Location library linked with the apk.
> 
> I can not 100% prove that using OM linked with GP location services doesn't do anything bad in the background, and would be interested to hear it from someone who has time and can monitor outgoing network connections, for example.

As said before, my biggest concern is not OM build (including the kayak feature), but using the PlayStore itself.

**Comment by The-magic-wand** [comment]
Dear @biodranik,
I contacted you via mail but unfortunately I had no response. Are you ignoring this messages?

I agree with user @muralito.

Why:
1. Downloading APK manually is not convenient because you will not have automatic updates and will have to check this manually.
2. Using google evil corp services is way out of privacy because library is proprietary and we have no 100% proof that it can’t do anything malicious.
3. I trust F-Droid developers because they build apk themselves, so we have 100% guarantee that this build is simmilar to source code.

**Comment by mnalis** [comment]
> I can not 100% prove that using OM linked with GP location services doesn't do anything bad in the background, and would be interested to hear it from someone who has time and can monitor outgoing network connections, for example.

@biodranik what would be hypothetical result if someone were to find that GMS or related services upon starting OM (or OM process itself) were to attempt network connection to google (or some other non-OM-controlled) network location?

OM suing Google? OM removing GMS support in all builds (not just f-droid ones)? OM just saying "oh well it is what it is" ? Something else?

While I only install f-droid app on my main phone, I have testing phone that I could put towards some testing and could install OM github release (which is the same as Google Play release?) and see how it behaves, if there is a realistic chance that all that work would result in some real benefit to user privacy.

**Comment by pedrodelbosque** [comment]
Organic Maps was created to give users an alternative to privacy-invasive big brother mapping apps. Childish "no! im right!" fights only hurt the end-users. Let's be real, F-droid users are a small fraction of all installs. Simply patch referral links out of the f-droid version. Or add "Referral Links" Setting, which is turned on by default everywhere but f-droid. Stop thinking about profits for a moment and remember why Organic Maps exists in the first place.

**Comment by mnalis** [comment]
> Simply patch referral links out of the f-droid version

@pedrodelbosque you (or anyone else) are free to submit MR for https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/app.organicmaps.yml. You can use [this example app](https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/de.westnordost.streetcomplete.yml) with [this patch](https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/de.westnordost.streetcomplete/streetcomplete-remove-arcore.patch) to get an idea how to do it. While writing the initial patch is relatively simple as you note, the main disadvantage is that you (or someone else) need to _indefinitely_ keep updating the patch to current upstream codebase, or the OrganicMaps might fail to build/publish newer versions on F-droid, which is why I personally don't find that option particularly attractive. (F-droid does that for their OsmAnd build for example, and it is hard and boring work that I cannot thank enough for volunteers who do it again and again) 

> Or add "Referral Links" Setting, which is turned on by default everywhere but f-droid

Yes, that would be significantly better solution, and was started in PR https://github.com/organicmaps/organicmaps/pull/6720. It's been abandoned since however, so I've [offered to pick it up](https://github.com/organicmaps/organicmaps/pull/7262#issuecomment-1920233499), if Organic Maps leadership would accept patch to add such preference. I'm still waiting for their answer, though (and judging by past experience, am getting worried if I'll be graced with an answer).

> Stop thinking about profits for a moment 

I wouldn't go so far, though. While there are certainly more important things in life then mere profits, profits are important for current OrganicMaps structure - if they can't pay for people, many developers wouldn't be able to work on OrganicMaps (or at least not as much), and in (theoretical) worst case scenario the project might even get abandoned (or significantly slowed down). 

The profits from that controversial feature however seem to have been minuscule, [totaling at about $100](https://github.com/organicmaps/organicmaps/pull/7262#issue-2104257106) altogether in about the most profitable season (Christmas + New Year), so I cannot fathom why Organic Maps leadership seems to continue to insist on keeping it, even if they [previously promised to remove Kayak feature if the money earned wasn't significant](https://github.com/organicmaps/organicmaps/pull/6523#issuecomment-1815997366).

> and remember why Organic Maps exists in the first place.

That I agree with absolutely. Be **proactive** in protecting user privacy. Actively help them to avoid being abused instead of saying _"it's their problem"_. Don't throw ads on them. Don't resell their data. In short - always be the friend ***of your FOSS OM user***, and **not** the friend of the big privacy-abusing companies like Kayak. Even if the the latter might pay better.

> Organic Maps was created to give users an alternative to privacy-invasive big brother mapping apps

That is how I remember it starting too, but I'm no longer sure if the same vision is still shared by Organic Maps leadership (although some developers at least seem to be very supportive of pro-user-privacy original vision - thank you for your efforts, you know who you are :heart: ).

**Comment by licaon-kter** [comment]
@biodranik 
> show me the rules

Why is https://github.com/organicmaps/issues/7000 inaccessible?

**Comment by pastk** [comment]
> 
> Why is https://github.com/organicmaps/issues/7000 inaccessible?

The url is wrong, `https://github.com/organicmaps/organicmaps/issues/7000` works: https://github.com/organicmaps/organicmaps/issues/7000
